backend/server/index.ts

```diff
@@ -0,0 +1,16 @@
+import express from "express";
+import * as bodyParser from "body-parser";
+
+const app = express();
+export const jsonParser = bodyParser.json()
```

Comment:
take a look at the "Express/Connect top-level generic" section. Better to connect json-parser for the whole express app in one place at server.ts